<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Operation</class>
 <widget class="QWidget" name="Operation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>1035</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QGridLayout" name="gridLayout_8" rowstretch="1,6,12,18">
      <item row="0" column="0">
       <widget class="QCheckBox" name="check_Bright">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>明场</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="check_Flu">
        <property name="text">
         <string>荧光</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_8">
        <property name="title">
         <string>拍摄</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_11">
         <item row="1" column="0" colspan="2">
          <widget class="QGroupBox" name="groupBox_10">
           <property name="title">
            <string>浏览</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QPushButton" name="btn_viewPicture">
              <property name="text">
               <string>浏览</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="btn_viewLast">
              <property name="text">
               <string>上一张</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="btn_viewNext">
              <property name="text">
               <string>下一张</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="btn_backToShot">
              <property name="text">
               <string>返回拍摄</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_11">
           <property name="title">
            <string>自动</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_12">
            <item row="0" column="0">
             <widget class="QPushButton" name="btn_shotSave">
              <property name="text">
               <string>拍摄并保存</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QGroupBox" name="groupBox_12">
           <property name="title">
            <string>手动</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_13">
            <item row="1" column="1">
             <widget class="QPushButton" name="btn_savePicture">
              <property name="text">
               <string>保存</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="btn_shot">
              <property name="text">
               <string>拍摄</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_13">
        <property name="title">
         <string>荧光设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_14" rowstretch="0,0,0,0,3">
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;亮度&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>对比度</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSlider" name="horizontalSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="spinBox_3"/>
         </item>
         <item row="3" column="1">
          <widget class="QSlider" name="horizontalSlider_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSlider" name="horizontalSlider_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QSpinBox" name="spinBox_5"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>激发光</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QSpinBox" name="spinBox_4"/>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="QGroupBox" name="groupBox_14">
           <property name="title">
            <string>镜头波长(nm)</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,3,3,3,4">
            <item>
             <widget class="QRadioButton" name="rbtn_510">
              <property name="text">
               <string>510</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbtn_540">
              <property name="text">
               <string>540</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbtn_560">
              <property name="text">
               <string>560</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbtn_580">
              <property name="text">
               <string>580</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbtn_All">
              <property name="text">
               <string>全光谱</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QGroupBox" name="groupBox_15">
           <property name="title">
            <string>激发光波长(nm)</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QRadioButton" name="radioButton_6">
              <property name="text">
               <string>425</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_7">
              <property name="text">
               <string>455</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_8">
              <property name="text">
               <string>480</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_9">
              <property name="text">
               <string>全光谱</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_16">
        <property name="title">
         <string>色阶调整</string>
        </property>
        <widget class="QSlider" name="horizontalSlider_5">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>40</y>
           <width>321</width>
           <height>16</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QCustomPlot" name="widget_3" native="true">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>70</y>
           <width>261</width>
           <height>231</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="colorbarLabel">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>70</y>
           <width>41</width>
           <height>211</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;255&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>290</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
